webapp/components/DateTimeRange/DateTimeRange.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <ds-text class="date-time-range" align="left" color="soft" :size="size">
+    <div>
+      <div>
+        <base-icon name="calendar" data-test="calendar" />
+        {{ getStartDateString }}
+      </div>
+      <div>
+        <base-icon name="clock" data-test="calendar" />
+        {{
+          getStartTimeString + (this.endDateAsDate && isSameDay ? '&mdash;' + getEndTimeString : '')
+        }}
+      </div>
+    </div>
+    <template v-if="!isSameDay">
+      &nbsp;&mdash;&nbsp;
+      <div>
+        <div>
+          <base-icon name="calendar" data-test="calendar" />
+          {{ getEndDateString }}
+        </div>
+        <div>
+          <base-icon name="clock" data-test="calendar" />
+          {{ getEndTimeString }}
+        </div>
+      </div>
+    </template>
+  </ds-text>
+</template>
+
+<script>
+import { format, isSameDay, isSameYear } from 'date-fns'
+
+export default {
+  name: 'DateTimeRange',
+  props: {
+    /**
+     * The size used for the text.
+     * @options small|base|large|x-large|xx-large|xxx-large
+     */
+    size: {
+      type: String,
+      default: null,
+      validator: (value) => {
+        return value.match(/(small|base|large|x-large|xx-large|xxx-large)/)
+      },
+    },
+    startDate: {
+      type: String,
+      require: true,
+    },
+    endDate: {
+      type: String,
+      default: null,
+    },
+  },
+  computed: {
+    startDateAsDate() {
+      return new Date(this.startDate)
+    },
+    endDateAsDate() {
+      return this.endDate ? new Date(this.endDate) : null
+    },
+    isSameDay() {
```

Comment:
Do you use the same name as the imported  `isSameDate` method on purpose? This might be confusing. Sure that there are no problems?